pkg/virt-launcher/virtwrap/cli/libvirt.go

```diff
@@ -264,7 +264,7 @@ func (l *LibvirtConnection) GetAllDomainStats(statsTypes libvirt.DomainStatsType
 	return domStats, nil
 }
 
-func (l *LibvirtConnection) GetDomainStats(statsTypes libvirt.DomainStatsTypes, flags libvirt.ConnectGetAllDomainStatsFlags) ([]*stats.DomainStats, error) {
+func (l *LibvirtConnection) GetDomainStats(statsTypes libvirt.DomainStatsTypes, migrateJobInfo *stats.DomainJobInfo, flags libvirt.ConnectGetAllDomainStatsFlags) ([]*stats.DomainStats, error) {
```

Comment:
There's no actual reason to pass this as an argument as you can fetch is from the `Domain`:

```go
domJobInfo, err := domStat.Domain.GetJobInfo()
```

Reply:
Would you mind if I change this in a follow up PR? we need to add another metric for the migration rate and I think this may make it easier, but if it won't I will remove this.

Reply:
Can you please explain? I can't really see how this could make life easier

Reply:
removed, if I will need it I will add it later.

Reply:
added it back, there is problem with the dirty rate metrics, if you want to suggest a better implementation, please do - and I will rework it in the next PR.

Reply:
I still can't understand the problem. Can you help me understand?